Dockerfile

```diff
@@ -51,16 +41,18 @@ ENV PREFIX="/opt/tenzir/vast" \
 ARG VAST_BUILD_OPTIONS
 
 RUN cmake -B build -G Ninja \
-      ${VAST_BUILD_OPTIONS} \
       -D CMAKE_INSTALL_PREFIX:STRING="$PREFIX" \
       -D CMAKE_BUILD_TYPE:STRING="Release" \
+      -D VAST_ENABLE_AVX_INSTRUCTIONS:BOOL="OFF" \
+      -D VAST_ENABLE_AVX2_INSTRUCTIONS:BOOL="OFF" \
       -D VAST_ENABLE_UNIT_TESTS:BOOL="OFF" \
       -D VAST_ENABLE_DEVELOPER_MODE:BOOL="OFF" \
       -D VAST_ENABLE_BUNDLED_CAF:BOOL="ON" \
       -D VAST_ENABLE_BUNDLED_SIMDJSON:BOOL="ON" \
       -D VAST_ENABLE_MANPAGES:BOOL="OFF" \
       -D VAST_ENABLE_PYTHON_BINDINGS_DEPENDENCIES:BOOL="ON" \
-      -D VAST_PLUGINS:STRING="plugins/*" && \
+      -D VAST_PLUGINS:STRING="plugins/*" \
```

Comment:
Should this be removed in favor of the new default that includes the plugins in `contrib/`? If not, then we probably shouldn't copy `contrib/` into the Dockerfile in the first place.